files/build_templates/init_cfg.json.j2

```diff
@@ -14,19 +16,18 @@
             "{{crm_res}}_threshold_type": "percentage",
             "{{crm_res}}_low_threshold": "70",
             "{{crm_res}}_high_threshold": "85"{% if not loop.last %},{% endif -%}
-{% endfor %}
+{%- endfor %}
         }
     },
     "FEATURE": {
-{%- for feature in ["sflow", "telemetry"] %}
-        "{{feature}}": {
+{%- for feature in optional_feature_list %}
+        "{{ feature }}": {
             "status": "disabled"
         }{% if not loop.last %},{% endif -%}
-{% endfor %}
+{%- endfor %}
     },
     "CONTAINER_FEATURE": {
-{%- for container in ["bgp", "database", "dhcp_relay", "lldp", "pmon", "radv", "restapi", "sflow",
-                      "snmp", "swss", "syncd", "teamd", "telemetry"] %}
+{%- for container in docker_containers_list %}
```

Comment:
It appears that Docker containers in the `optional_feature_list` will not get populated here in the `CONTAINER_FEATURE` table.

Also, please note that we are thinking of expanding the concept of a "FEATURE" to include any type of service, whether or not it spawns a container. This way all services can be enabled or disabled from the SONiC CLI. Thus, this logic would need to change.

Reply:
@jleveque docker_containers_list is based on $($*_DOCKERS) which includes all optional and non optional docker containers.

Regarding second comment, I can rename SONIC_OPTIONAL_DOCKER_CONTAINERS to SONIC_OPTIONAL_SERVICES, so when you will need to add non-docker service you can append service name to that list and that will work for these services too.